app/views/hq/partners/_form.html.erb

```diff
@@ -29,7 +29,7 @@
   <div class="form-group">
     <%= f.submit class: 'btn btn-primary' %>
     <%= link_to('Cancel',
-                (@partner.id ? hq_partner_path(@partner.id) : hq_partners_path),
+                (@partner_facade.partner.id ? hq_partner_path(@partner_facade.partner) : hq_partners_path),
                 class: 'btn btn-default btn-cancel', role: 'button') %>
```

Comment:
I guess this link path logic should go inside the facade too?
And then use it like `link_to('Cancel', @partner_facade.cancel_button_path, ... )`